eng/scripts/python/apiview-syncdb/sync_cosmosdb.py

```diff
@@ -25,13 +26,14 @@
 BACKUP_CONTAINER = "backups"
 BLOB_NAME_PATTERN ="cosmos/{0}/{1}"
 
+# Create a AzurePowerShellCredential()
+credentials = AzurePowerShellCredential()
```

Comment:
This will work fine for the pipeline but if anyone runs this locally we might want to consider az cli credential as well.